template/pages/home.js

```diff
@@ -10,6 +11,7 @@ module.exports = (store, actions) => {
   return html`
     <div>
       <app-header color=${store.color} onclick=${advanceColor} />
+      <no-js-warning />
```

Comment:
instead of having an entire route dedicated to when js is disabled, including a component that is a `<noscript>` tag.

this should be less confusing and easier to implement in future webapps